wallet_transactions/userapi/views.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import generics
+from rest_framework.response import Response
+from .serializers import RegisterSerializer, UserSerializer
+
+
+class RegisterApi(generics.GenericAPIView):
+    """Create view for register user with method post"""
```

Comment:
have you tested register, login, send requests with a token?

Reply:
Yes, it is work